src/Widgets/Presets/DefaultCheckoutPreset.php

```diff
@@ -16,72 +15,105 @@ class DefaultCheckoutPreset implements ContentPreset
         $this->createContactWishWidget();
         $this->createShippingPrivacyCheckWidget();
         $this->createGtcCheckWidget();
+        $this->createAsteriks();
         $this->createBasketWidget();
+        $this->createSeparatorWidget('second');
         $this->createCouponWidget();
+        $this->createSeparatorWidget('second');
+        $this->createStickyContainer();
         $this->createBasketTotalsWidget();
         $this->createPlaceOrderWidget();
         
         return $this->preset->toArray();
     }
     
-    private function createAddressWidget()
+    private function createAddressWidget($type = '1')
     {
-        $this->twoColumnWidget->createChild('first','Ceres::AddressWidget');
+        $addressWidget = $this->twoColumnWidget->createChild('first','Ceres::AddressWidget')->withSetting('addressType', $type);
+        
+        if($type == '1')
+        {
+            $addressWidget
+                ->withSetting('addressFieldsInvoiceDE', $this->ceresConfig->addresses->billingAddressShow)
+                ->withSetting('addressRequiredFieldsInvoiceDE', $this->ceresConfig->addresses->billingAddressRequire)
+                ->withSetting('addressFieldsInvoiceGB', $this->ceresConfig->addresses->billingAddressShow_en)
+                ->withSetting('addressRequiredFieldsInvoiceGB', $this->ceresConfig->addresses->billingAddressRequire_en);
+        }
+        elseif($type == '2')
+        {
+            $addressWidget
+                ->withSetting('addressFieldsShippingDE', $this->ceresConfig->addresses->deliveryAddressShow)
+                ->withSetting('addressRequiredFieldsShippingDE', $this->ceresConfig->addresses->deliveryAddressRequire)
+                ->withSetting('addressFieldsShippingGB', $this->ceresConfig->addresses->deliveryAddressShow_en)
+                ->withSetting('addressRequiredFieldsShippingGB', $this->ceresConfig->addresses->deliveryAddressRequire_en);
+        }
+        
+        
     }
     
     private function createBasketTotalsWidget()
     {
-        $this->twoColumnWidget->createChild('second','Ceres::BasketTotalsWidget');
+        $this->stickyContainer->createChild('sticky','Ceres::BasketTotalsWidget')
+            ->withSetting('visibleFields', ["basketValueNet", "basketValueGross", "rebate", "shippingCostsNet", "shippingCostsGross", "totalSumNet", "coupon", "vats", "totalSumGross", "couponDiscount", "openAmount"]);
     }
     
     private function createBasketWidget()
     {
-        $this->twoColumnWidget->createChild('second','Ceres::BasketWidget');
+        $this->twoColumnWidget->createChild('second','Ceres::BasketWidget')
+            ->withSetting('appearance', 'primary');
     }
     
     private function createContactWishWidget()
     {
-        $this->twoColumnWidget->createChild('first','Ceres::ContactWishWidget');
+        $this->twoColumnWidget->createChild('first','Ceres::ContactWishWidget')
+            ->withSetting('hinttext', '');
```

Comment:
Das wurde in einem anderen PR noch angepasst. Die Variable heißt jetzt "hintText"